src/Infrastructure/Sofort/Transfer/Request.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure\Sofort\Transfer;
+
+use WMDE\Euro\Euro;
+
+class Request {
+
+	/**
+	 * @var Euro
+	 */
+	private $amount;
+	/**
+	 * @var string
+	 */
+	private $currencyCode = '';
+	/**
+	 * @var string
+	 */
+	private $successUrl = '';
+	/**
+	 * @var string
+	 */
+	private $abortUrl = '';
+	/**
+	 * @var string
+	 */
+	private $notificationUrl = '';
+	/**
+	 * @var string[]
+	 */
+	private $reasons = [ '', '' ];
+
+	/**
+	 * @return Euro
+	 */
+	public function getAmount(): Euro {
+		return $this->amount;
+	}
+
+	/**
+	 * @param Euro $amount
+	 */
+	public function setAmount( Euro $amount ): void {
```

Comment:
They are not really, but they are part of my workflow for value objects - specifying the properties and using PHPStorm to auto-generate the accessors.
You could argue that their presence increases the chance for inconsistency whenever either the method signature or the phpdoc change independently of the other.. do you feel strongly about this?
Given - I believe - there is no real rule not to have them, what is the consensus on this?
@gbirke @KaiNissen